gmd_config.py

```
@@ -0,0 +1,88 @@
+def _load_config_dict() -> Tuple[Dict[str, Any], Optional[Path]]:
+    candidates: list[Path] = []
+    env_path = os.environ.get(CONFIG_ENV)
+    if env_path:
+        candidates.append(Path(env_path))
+    candidates.extend(DEFAULT_PATHS)
+    for path in candidates:
+        try:
+            if path.exists():
+                return json.loads(path.read_text(encoding="utf-8")), path
+        except Exception:
+            continue
```

Ruff (0.15.9)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)